### ISSUE TITLE
compilation errors when attempting to install via cargo

### ISSUE THREAD

**Issue by masukomi** [Canop/broot#209]
**Describe the bug**
running ` cargo install broot --version 0.13.2 --verbose` results in compilation errors 

**To Reproduce**
` cargo install broot --version 0.13.2 --verbose` 

**Expected behavior**
compiles and installs

**Configuration (please complete the following information):**
 - OS: macos
 - Version v10.15.3 (catalina)
- Relevant configuration items: 🤷🏻‍♀️

**Additional context**

```

error: enum variants on type aliases are experimental
  --> /Users/masukomi/.cargo/registry/src/github.com-1ecc6299db9ec823/broot-0.13.2/src/shell_install/mod.rs:63:31
   |
63 |             "undefined" => Ok(Self::Undefined),
   |                               ^^^^^^^^^^^^^^^

error: enum variants on type aliases are experimental
  --> /Users/masukomi/.cargo/registry/src/github.com-1ecc6299db9ec823/broot-0.13.2/src/shell_install/mod.rs:64:29
   |
64 |             "refused" => Ok(Self::Refused),
   |                             ^^^^^^^^^^^^^

error: enum variants on type aliases are experimental
  --> /Users/masukomi/.cargo/registry/src/github.com-1ecc6299db9ec823/broot-0.13.2/src/shell_install/mod.rs:65:31
   |
65 |             "installed" => Ok(Self::Installed),
   |                               ^^^^^^^^^^^^^^^

error: enum variants on type aliases are experimental
  --> /Users/masukomi/.cargo/registry/src/github.com-1ecc6299db9ec823/broot-0.13.2/src/shell_install/mod.rs:89:13
   |
89 |             Self::Refused => {
   |             ^^^^^^^^^^^^^

error: enum variants on type aliases are experimental
  --> /Users/masukomi/.cargo/registry/src/github.com-1ecc6299db9ec823/broot-0.13.2/src/shell_install/mod.rs:93:13
   |
93 |             Self::Installed => {
   |             ^^^^^^^^^^^^^^^

error: enum variants on type aliases are experimental
  --> /Users/masukomi/.cargo/registry/src/github.com-1ecc6299db9ec823/broot-0.13.2/src/task_sync.rs:23:13
   |
23 |             Self::Done(_) => true,
   |             ^^^^^^^^^^^^^

error: enum variants on type aliases are experimental
  --> /Users/masukomi/.cargo/registry/src/github.com-1ecc6299db9ec823/broot-0.13.2/src/task_sync.rs:29:13
   |
29 |             Self::NotComputed => true,
   |             ^^^^^^^^^^^^^^^^^

error: enum variants on type aliases are experimental
  --> /Users/masukomi/.cargo/registry/src/github.com-1ecc6299db9ec823/broot-0.13.2/src/task_sync.rs:35:13
   |
35 |             Self::None => true,
   |             ^^^^^^^^^^

error: aborting due to 8 previous errors

error: failed to compile `broot v0.13.2`, intermediate artifacts can be found at `/var/folders/8_/1jftc8kn6h55st0j7_gjkc0m0000gn/T/cargo-installKGPhW1`

Caused by:
  Could not compile `broot`.

Caused by:
  process didn't exit successfully: `rustc --edition=2018 --crate-name broot /Users/masukomi/.cargo/registry/src/github.com-1ecc6299db9ec823/broot-0.13.2/src/lib.rs --color always --crate-type lib --emit=dep-info,metadata,link -C opt-level=3 -C codegen-units=1 -C metadata=0314b09032199497 -C extra-filename=-0314b09032199497 --out-dir /var/folders/8_/1jftc8kn6h55st0j7_gjkc0m0000gn/T/cargo-installKGPhW1/release/deps -L dependency=/var/folders/8_/1jftc8kn6h55st0j7_gjkc0m0000gn/T/cargo-installKGPhW1/release/deps --extern chrono=/var/folders/8_/1jftc8kn6h55st0j7_gjkc0m0000gn/T/cargo-installKGPhW1/release/deps/libchrono-78e557f69c7d5253.rlib --extern clap=/var/folders/8_/1jftc8kn6h55st0j7_gjkc0m0000gn/T/cargo-installKGPhW1/release/deps/libclap-c52e7fe6c1e171ff.rlib --extern crossbeam=/var/folders/8_/1jftc8kn6h55st0j7_gjkc0m0000gn/T/cargo-installKGPhW1/release/deps/libcrossbeam-56923736f4fa5f67.rlib --extern crossterm=/var/folders/8_/1jftc8kn6h55st0j7_gjkc0m0000gn/T/cargo-installKGPhW1/release/deps/libcrossterm-dd9ce988244854d3.rlib --extern custom_error=/var/folders/8_/1jftc8kn6h55st0j7_gjkc0m0000gn/T/cargo-installKGPhW1/release/deps/libcustom_error-ba38a0d578829bcb.rlib --extern directories=/var/folders/8_/1jftc8kn6h55st0j7_gjkc0m0000gn/T/cargo-installKGPhW1/release/deps/libdirectories-c550d5a1d7dd3a64.rlib --extern git2=/var/folders/8_/1jftc8kn6h55st0j7_gjkc0m0000gn/T/cargo-installKGPhW1/release/deps/libgit2-1476cdaf44e1d5c3.rlib --extern glob=/var/folders/8_/1jftc8kn6h55st0j7_gjkc0m0000gn/T/cargo-installKGPhW1/release/deps/libglob-c2df5c47e7a9afe7.rlib --extern id_arena=/var/folders/8_/1jftc8kn6h55st0j7_gjkc0m0000gn/T/cargo-installKGPhW1/release/deps/libid_arena-8da9271f4311606b.rlib --extern lazy_regex=/var/folders/8_/1jftc8kn6h55st0j7_gjkc0m0000gn/T/cargo-installKGPhW1/release/deps/liblazy_regex-fc8ba510df098d87.rlib --extern lazy_static=/var/folders/8_/1jftc8kn6h55st0j7_gjkc0m0000gn/T/cargo-installKGPhW1/release/deps/liblazy_static-eca90f48542cf57f.rlib --extern log=/var/folders/8_/1jftc8kn6h55st0j7_gjkc0m0000gn/T/cargo-installKGPhW1/release/deps/liblog-4679bcaaccf3c936.rlib --extern minimad=/var/folders/8_/1jftc8kn6h55st0j7_gjkc0m0000gn/T/cargo-installKGPhW1/release/deps/libminimad-15abf6462125c053.rlib --extern open=/var/folders/8_/1jftc8kn6h55st0j7_gjkc0m0000gn/T/cargo-installKGPhW1/release/deps/libopen-5e7ef1aef50d93cc.rlib --extern pathdiff=/var/folders/8_/1jftc8kn6h55st0j7_gjkc0m0000gn/T/cargo-installKGPhW1/release/deps/libpathdiff-b69736cc3663867f.rlib --extern regex=/var/folders/8_/1jftc8kn6h55st0j7_gjkc0m0000gn/T/cargo-installKGPhW1/release/deps/libregex-6c6ace9533936189.rlib --extern simplelog=/var/folders/8_/1jftc8kn6h55st0j7_gjkc0m0000gn/T/cargo-installKGPhW1/release/deps/libsimplelog-5bc362a4cfda1efb.rlib --extern termimad=/var/folders/8_/1jftc8kn6h55st0j7_gjkc0m0000gn/T/cargo-installKGPhW1/release/deps/libtermimad-14de7fe7de4042fa.rlib --extern toml=/var/folders/8_/1jftc8kn6h55st0j7_gjkc0m0000gn/T/cargo-installKGPhW1/release/deps/libtoml-b27e74480da7b2b1.rlib --extern umask=/var/folders/8_/1jftc8kn6h55st0j7_gjkc0m0000gn/T/cargo-installKGPhW1/release/deps/libumask-2ac095d64cd0f7ba.rlib --extern users=/var/folders/8_/1jftc8kn6h55st0j7_gjkc0m0000gn/T/cargo-installKGPhW1/release/deps/libusers-ee83af3d58b6c65b.rlib --cap-lints allow -L native=/var/folders/8_/1jftc8kn6h55st0j7_gjkc0m0000gn/T/cargo-installKGPhW1/release/build/libgit2-sys-f6d837567e317e58/out/build` (exit code: 1)

```

**Comment by Canop** [comment]
You might have an old rust compiler. Please upgrade it.
If you installed it with rustup, run `rustup update`.

**Comment by masukomi** [comment]
still fails after `rustup update` although the error is different.


```
   Compiling broot v0.13.2
     Running `rustc --edition=2018 --crate-name build_script_build /Users/masukomi/.cargo/registry/src/github.com-1ecc6299db9ec823/broot-0.13.2/build.rs --error-format=json --json=diagnostic-rendered-ansi --crate-type bin --emit=dep-info,link -C opt-level=3 -C codegen-units=1 -C metadata=67782f71d4f96d30 -C extra-filename=-67782f71d4f96d30 --out-dir /var/folders/8_/1jftc8kn6h55st0j7_gjkc0m0000gn/T/cargo-installOPlvge/release/build/broot-67782f71d4f96d30 -L dependency=/var/folders/8_/1jftc8kn6h55st0j7_gjkc0m0000gn/T/cargo-installOPlvge/release/deps --extern clap=/var/folders/8_/1jftc8kn6h55st0j7_gjkc0m0000gn/T/cargo-installOPlvge/release/deps/libclap-8fe1e7f82b4fb8e9.rlib --cap-lints allow`
     Running `/var/folders/8_/1jftc8kn6h55st0j7_gjkc0m0000gn/T/cargo-installOPlvge/release/build/broot-67782f71d4f96d30/build-script-build`
     Running `rustc --edition=2018 --crate-name broot /Users/masukomi/.cargo/registry/src/github.com-1ecc6299db9ec823/broot-0.13.2/src/lib.rs --error-format=json --json=diagnostic-rendered-ansi --crate-type lib --emit=dep-info,metadata,link -C opt-level=3 -C codegen-units=1 -C metadata=fe34d17c17ab300b -C extra-filename=-fe34d17c17ab300b --out-dir /var/folders/8_/1jftc8kn6h55st0j7_gjkc0m0000gn/T/cargo-installOPlvge/release/deps -L dependency=/var/folders/8_/1jftc8kn6h55st0j7_gjkc0m0000gn/T/cargo-installOPlvge/release/deps --extern chrono=/var/folders/8_/1jftc8kn6h55st0j7_gjkc0m0000gn/T/cargo-installOPlvge/release/deps/libchrono-9df426afef7c3212.rmeta --extern clap=/var/folders/8_/1jftc8kn6h55st0j7_gjkc0m0000gn/T/cargo-installOPlvge/release/deps/libclap-8fe1e7f82b4fb8e9.rmeta --extern crossbeam=/var/folders/8_/1jftc8kn6h55st0j7_gjkc0m0000gn/T/cargo-installOPlvge/release/deps/libcrossbeam-5ce30f3295ad2094.rmeta --extern crossterm=/var/folders/8_/1jftc8kn6h55st0j7_gjkc0m0000gn/T/cargo-installOPlvge/release/deps/libcrossterm-9d62a6b7ebe78c14.rmeta --extern custom_error=/var/folders/8_/1jftc8kn6h55st0j7_gjkc0m0000gn/T/cargo-installOPlvge/release/deps/libcustom_error-48f2db70b7589835.rmeta --extern directories=/var/folders/8_/1jftc8kn6h55st0j7_gjkc0m0000gn/T/cargo-installOPlvge/release/deps/libdirectories-d209fa4bfe721144.rmeta --extern git2=/var/folders/8_/1jftc8kn6h55st0j7_gjkc0m0000gn/T/cargo-installOPlvge/release/deps/libgit2-bdd82a5aba5ee6b7.rmeta --extern glob=/var/folders/8_/1jftc8kn6h55st0j7_gjkc0m0000gn/T/cargo-installOPlvge/release/deps/libglob-f44b35c13ea481d6.rmeta --extern id_arena=/var/folders/8_/1jftc8kn6h55st0j7_gjkc0m0000gn/T/cargo-installOPlvge/release/deps/libid_arena-4ce2c174cdd82a3f.rmeta --extern lazy_regex=/var/folders/8_/1jftc8kn6h55st0j7_gjkc0m0000gn/T/cargo-installOPlvge/release/deps/liblazy_regex-b3396738d4cfa2e5.rmeta --extern lazy_static=/var/folders/8_/1jftc8kn6h55st0j7_gjkc0m0000gn/T/cargo-installOPlvge/release/deps/liblazy_static-edcc7eee37b83f6d.rmeta --extern log=/var/folders/8_/1jftc8kn6h55st0j7_gjkc0m0000gn/T/cargo-installOPlvge/release/deps/liblog-579306de5c07acbd.rmeta --extern minimad=/var/folders/8_/1jftc8kn6h55st0j7_gjkc0m0000gn/T/cargo-installOPlvge/release/deps/libminimad-b90a427b05d39214.rmeta --extern open=/var/folders/8_/1jftc8kn6h55st0j7_gjkc0m0000gn/T/cargo-installOPlvge/release/deps/libopen-84964569e514d0b2.rmeta --extern pathdiff=/var/folders/8_/1jftc8kn6h55st0j7_gjkc0m0000gn/T/cargo-installOPlvge/release/deps/libpathdiff-89dc4e47c9a109d1.rmeta --extern regex=/var/folders/8_/1jftc8kn6h55st0j7_gjkc0m0000gn/T/cargo-installOPlvge/release/deps/libregex-2d64965bcdd14846.rmeta --extern simplelog=/var/folders/8_/1jftc8kn6h55st0j7_gjkc0m0000gn/T/cargo-installOPlvge/release/deps/libsimplelog-b4c383c055229b88.rmeta --extern termimad=/var/folders/8_/1jftc8kn6h55st0j7_gjkc0m0000gn/T/cargo-installOPlvge/release/deps/libtermimad-cf34b59f4c1b8e6b.rmeta --extern toml=/var/folders/8_/1jftc8kn6h55st0j7_gjkc0m0000gn/T/cargo-installOPlvge/release/deps/libtoml-094ec33257a17fc3.rmeta --extern umask=/var/folders/8_/1jftc8kn6h55st0j7_gjkc0m0000gn/T/cargo-installOPlvge/release/deps/libumask-d2acacced596d83f.rmeta --extern users=/var/folders/8_/1jftc8kn6h55st0j7_gjkc0m0000gn/T/cargo-installOPlvge/release/deps/libusers-7f05206cc8b5ce8c.rmeta --cap-lints allow -L native=/var/folders/8_/1jftc8kn6h55st0j7_gjkc0m0000gn/T/cargo-installOPlvge/release/build/libgit2-sys-8abf9ebf62747438/out/build`
error[E0023]: this pattern has 2 fields, but the corresponding tuple variant has 3 fields
   --> /Users/masukomi/.cargo/registry/src/github.com-1ecc6299db9ec823/broot-0.13.2/src/commands.rs:190:13
    |
190 |             Event::Click(x, y) => {
    |             ^^^^^^^^^^^^^^^^^^ expected 3 fields, found 2
    |
   ::: /Users/masukomi/.cargo/registry/src/github.com-1ecc6299db9ec823/termimad-0.8.16/src/events/event.rs:15:5
    |
15  |     Click(u16, u16, KeyModifiers),
    |     ----------------------------- tuple variant defined here

error: aborting due to previous error

For more information about this error, try `rustc --explain E0023`.
error: could not compile `broot`.

Caused by:
  process didn't exit successfully: `rustc --edition=2018 --crate-name broot /Users/masukomi/.cargo/registry/src/github.com-1ecc6299db9ec823/broot-0.13.2/src/lib.rs --error-format=json --json=diagnostic-rendered-ansi --crate-type lib --emit=dep-info,metadata,link -C opt-level=3 -C codegen-units=1 -C metadata=fe34d17c17ab300b -C extra-filename=-fe34d17c17ab300b --out-dir /var/folders/8_/1jftc8kn6h55st0j7_gjkc0m0000gn/T/cargo-installOPlvge/release/deps -L dependency=/var/folders/8_/1jftc8kn6h55st0j7_gjkc0m0000gn/T/cargo-installOPlvge/release/deps --extern chrono=/var/folders/8_/1jftc8kn6h55st0j7_gjkc0m0000gn/T/cargo-installOPlvge/release/deps/libchrono-9df426afef7c3212.rmeta --extern clap=/var/folders/8_/1jftc8kn6h55st0j7_gjkc0m0000gn/T/cargo-installOPlvge/release/deps/libclap-8fe1e7f82b4fb8e9.rmeta --extern crossbeam=/var/folders/8_/1jftc8kn6h55st0j7_gjkc0m0000gn/T/cargo-installOPlvge/release/deps/libcrossbeam-5ce30f3295ad2094.rmeta --extern crossterm=/var/folders/8_/1jftc8kn6h55st0j7_gjkc0m0000gn/T/cargo-installOPlvge/release/deps/libcrossterm-9d62a6b7ebe78c14.rmeta --extern custom_error=/var/folders/8_/1jftc8kn6h55st0j7_gjkc0m0000gn/T/cargo-installOPlvge/release/deps/libcustom_error-48f2db70b7589835.rmeta --extern directories=/var/folders/8_/1jftc8kn6h55st0j7_gjkc0m0000gn/T/cargo-installOPlvge/release/deps/libdirectories-d209fa4bfe721144.rmeta --extern git2=/var/folders/8_/1jftc8kn6h55st0j7_gjkc0m0000gn/T/cargo-installOPlvge/release/deps/libgit2-bdd82a5aba5ee6b7.rmeta --extern glob=/var/folders/8_/1jftc8kn6h55st0j7_gjkc0m0000gn/T/cargo-installOPlvge/release/deps/libglob-f44b35c13ea481d6.rmeta --extern id_arena=/var/folders/8_/1jftc8kn6h55st0j7_gjkc0m0000gn/T/cargo-installOPlvge/release/deps/libid_arena-4ce2c174cdd82a3f.rmeta --extern lazy_regex=/var/folders/8_/1jftc8kn6h55st0j7_gjkc0m0000gn/T/cargo-installOPlvge/release/deps/liblazy_regex-b3396738d4cfa2e5.rmeta --extern lazy_static=/var/folders/8_/1jftc8kn6h55st0j7_gjkc0m0000gn/T/cargo-installOPlvge/release/deps/liblazy_static-edcc7eee37b83f6d.rmeta --extern log=/var/folders/8_/1jftc8kn6h55st0j7_gjkc0m0000gn/T/cargo-installOPlvge/release/deps/liblog-579306de5c07acbd.rmeta --extern minimad=/var/folders/8_/1jftc8kn6h55st0j7_gjkc0m0000gn/T/cargo-installOPlvge/release/deps/libminimad-b90a427b05d39214.rmeta --extern open=/var/folders/8_/1jftc8kn6h55st0j7_gjkc0m0000gn/T/cargo-installOPlvge/release/deps/libopen-84964569e514d0b2.rmeta --extern pathdiff=/var/folders/8_/1jftc8kn6h55st0j7_gjkc0m0000gn/T/cargo-installOPlvge/release/deps/libpathdiff-89dc4e47c9a109d1.rmeta --extern regex=/var/folders/8_/1jftc8kn6h55st0j7_gjkc0m0000gn/T/cargo-installOPlvge/release/deps/libregex-2d64965bcdd14846.rmeta --extern simplelog=/var/folders/8_/1jftc8kn6h55st0j7_gjkc0m0000gn/T/cargo-installOPlvge/release/deps/libsimplelog-b4c383c055229b88.rmeta --extern termimad=/var/folders/8_/1jftc8kn6h55st0j7_gjkc0m0000gn/T/cargo-installOPlvge/release/deps/libtermimad-cf34b59f4c1b8e6b.rmeta --extern toml=/var/folders/8_/1jftc8kn6h55st0j7_gjkc0m0000gn/T/cargo-installOPlvge/release/deps/libtoml-094ec33257a17fc3.rmeta --extern umask=/var/folders/8_/1jftc8kn6h55st0j7_gjkc0m0000gn/T/cargo-installOPlvge/release/deps/libumask-d2acacced596d83f.rmeta --extern users=/var/folders/8_/1jftc8kn6h55st0j7_gjkc0m0000gn/T/cargo-installOPlvge/release/deps/libusers-7f05206cc8b5ce8c.rmeta --cap-lints allow -L native=/var/folders/8_/1jftc8kn6h55st0j7_gjkc0m0000gn/T/cargo-installOPlvge/release/build/libgit2-sys-8abf9ebf62747438/out/build` (exit code: 1)
warning: build failed, waiting for other jobs to finish...
error: failed to compile `broot v0.13.2`, intermediate artifacts can be found at `/var/folders/8_/1jftc8kn6h55st0j7_gjkc0m0000gn/T/cargo-installOPlvge`

Caused by:
  build failed
```

**Comment by Canop** [comment]
@masukomi this one is due to an error of mine and is fixed in v0.13.3 (from yesterday).

**Comment by Bo98** [comment]
@Canop Can you push a [git tag](https://github.com/Canop/broot/releases) for v0.13.3?

**Comment by Canop** [comment]
@Bo98 OK, I'll try to do that tomorrow. Some things are a little more complicated today in France.

**Comment by Bo98** [comment]
I understand. Take care!

**Comment by Canop** [comment]
@Bo98 I just published the 0.13.4
https://github.com/Canop/broot/releases/tag/v0.13.4
